homeassistant/util/color.py

```diff
@@ -8,7 +8,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 HASS_COLOR_MAX = 500  # mireds (inverted)
-HASS_COLOR_MIN = 154
+HASS_COLOR_MIN = 111
```

Comment:
I guess the question is how do individual light systems handle commands for temperatures outside their supported range. Right now the values from color.py are being enforced by components/light/__init__.py, causing any commands to lifx outside this range to fail.